frontend/src/components/landing/Landing.tsx

```diff
@@ -191,16 +194,20 @@ const Landing = () => {
             </div>
           </div>
           <div className={styles.landingAnimation}>
-            <Lottie
-              options={{
-                loop: false,
-                autoplay: true,
-                animationData: landingAnimation,
-                rendererSettings: {
-                  preserveAspectRatio: 'xMidYMid slice',
-                },
-              }}
-            />
+            {isIE11 ? (
```

Comment:
The other lottie animations also don't work right? 
Is there a way to load just either the animation or the static images, instead of having to download both?

Reply:
Dynamic imports could do this, https://create-react-app.dev/docs/code-splitting/

Reply:
oh the other lottie animations work... just that the width is much larger than on chrome. why do the other lottie animations work but not the bird's?